src/credentials/sso/ssoAccessTokenProvider.ts

```diff
@@ -180,7 +181,7 @@ export class SsoAccessTokenProvider {
 
     private async registerClient(): Promise<ClientRegistration> {
         return this.oidc.registerClient({
-            clientName: `AWS Toolkit for VSCode`,
+            clientName: isCloud9() ? 'AWS Cloud9' : 'AWS Toolkit for VSCode',
```

Comment:
Need to to the `getIdeProperties().company` dance here. Might as well do it for both cases, though currently it will always be "AWS" for the non-Cloud9 case.